MAX_DIST: 0.65
MIN_DIST: 0.64

NUM_MEAN_SAMPLES: 10

MAX_GAP_DEPTH: 0.38
MIN_GAP_DEPTH: 0.20

MIN_GAP_LENGTH: 0.5
MAX_GAP_LENGTH: 1.0

RANGE_THRESHOLD: 0.05

BACKWARD_THRESHOLD_1: 0.8
BACKWARD_THRESHOLD_2: 0.15

LINEAR_SPEED: 0.45
ANGULAR_SPEED: 0.25

YAW_THRESHOLD: 0.4

STEERING_FIRST: 0.6
